pkg/webhook/webhook.go

```diff
@@ -90,6 +93,10 @@ func (w *Webhook) onSecretChange(_ string, secret *corev1.Secret) (*corev1.Secre
 	if err != nil {
 		return nil, err
 	}
+	w.azureDevops, err = azuredevops.New()
```

Comment:
I am surprised not to see any type of key here, unlike for other providers. Does auth work differently for Azure DevOps?

Reply:
https://github.com/go-playground/webhooks/pull/191 PR upstream